web/viewer.css

```diff
@@ -920,6 +920,24 @@ html[dir="rtl"] #viewOutline.toolbarButton::before {
   content: url(images/toolbarButton-search.png);
 }
 
+.toolbarButton.pdfSidebarNotification::after {
+  position: absolute;
+  display: inline-block;
+  top: 0px;
+  /* Create a filled circle, with a diameter of 10 pixels, using only CSS: */
+  content: '';
+  background-color: #8BE651;
+  height: 10px;
```

Comment:
I basically just picked a size, more or less arbitrarily, that wouldn't be too large while still being easily visible.
If we decrease the size, my only worry is that the notification could become too small to be easily noticeable. However, using a slightly brighter color would probably help with that.

Which of the following ones would you prefer?
https://www.dropbox.com/s/prv8vyuxagf5vln/notification-sizes.png?dl=0

Also, do you think that we should use a different tooltip for the `sidebarToggle` button when outline/attachments exists?
E.g. adding a new `toggle_sidebar_notification.title=Toggle Sidebar (document has outline/attachments)` localization string and using that here.